marshal_test.go

```
@@ -1300,3 +1300,5 @@ func TestCollectionNewWithErrorConsistentWithGoType(t *testing.T) {
 		}
 	}
 }
+
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(goimports)